.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,13 @@
-<!-- Nixpkgs has a lot of new incoming Pull Requests, but not enough people to review this constant stream. Even if you aren't a committer, we would appreciate reviews of other PRs, especially simple ones like package updates. Just testing the relevant package/service and leaving a comment saying what you tested, how you tested it and whether it worked would be great. List of open PRs: <https://github.com/NixOS/nixpkgs/pulls>, for more about reviewing contributions: <https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download/1/nixpkgs/manual.html#chap-reviewing-contributions>. Reviewing isn't mandatory, but it would help out a lot and reduce the average time-to-merge for all of us. Thanks a lot if you do! -->
+<!--
+We would appreciate your reviews of other PRs to help with the large amounts of
+pull request, especially simple package updates.
```

Comment:
```suggestion
We would appreciate your review of other Pull Requests, especially simple package updates,
to help distribute the burden.
```

Or something similar? It sounds weird to me to say "your reviews of other PRs to help with pull requests" -- either stick with the acronym, or don't (and the structure was just plain weird IMHO).

---

Also, related to the general structure: maybe it would be better to reflow this to fit 72-80 characters? I think it would look nicer, but if the current layout works fine for the `hub` use-case, feel free to disregard this.

Reply:
Fixed and reflowed.